src/utils/bytes.js

```diff
@@ -59,3 +107,17 @@ export function getCryptoRandomBytesArray(win, length) {
   win.crypto.getRandomValues(uint8array);
   return uint8array;
 }
+
+/**
+ * Generates an ArrayBuffer from the given string.
+ * @param {!string} string The String to be converted.
+ * @return {!ArrayBuffer}
+ */
+export function stringToArrayBuffer(string) {
+  const buffer = new ArrayBuffer(string.length * 2);
```

Comment:
You can find a discussion on this function here: https://developers.google.com/web/updates/2012/06/How-to-convert-ArrayBuffer-to-and-from-String. The *2 is due to the fact that you'll need two bytes per character.